src/components/common/ParentNavTopBar.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Typography, Menu, Dropdown } from 'antd';
+import { useHistory } from 'react-router-dom';
+import { useOktaAuth } from '@okta/okta-react';
+import { connect } from 'react-redux';
+import { global } from '../../state/actions';
+
+import parent_avatar from '../../assets/icons/parent_avatar.svg';
+
+const { Title } = Typography;
+
+const ParentMenu = props => {
+  const { push } = useHistory();
+  const { authService } = useOktaAuth();
+  const { clearUsers, ...rest } = props;
+
+  const switchUsers = e => {
+    clearUsers();
+    push('/');
+  };
+
+  return (
+    <Menu {...rest}>
+      <Menu.Item key="1" onClick={switchUsers}>
+        Change User
+      </Menu.Item>
+      <Menu.Item key="2" onClick={() => authService.logout()}>
+        Logout
+      </Menu.Item>
+    </Menu>
+  );
+};
+
+const ParentNavTopBar = props => {
+  return (
+    <nav className="parent-nav-top-bar" theme="light">
+      <a href="">
+        <Title className="title navbar-logo" style={{ margin: 0 }} level={1}>
+          STORY SQUAD
+        </Title>
+      </a>
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <span className="welcome-back-msg">
+          Welcome back, {props.parent && props.parent.name}
```

Comment:
I see parent being pulled from the store at the bottom of the page but I don't understand why props needs to be prepended to it here. When I go up a level to where ParentNavTopBar is rendered the only prop I see being passed in is 'selected'.

Reply:
![Screen Shot 2021-05-01 at 10 21 03 AM](https://user-images.githubusercontent.com/62610548/116790008-f47ab280-aa66-11eb-9edb-1a6d7ae17cb1.png)

Reply:
If I understand your question correctly: the reason I had to prepend 'props' to the parent is due to Redux Connect API mapping the object to props.  Without `props.`, I would have no way of accessing the obj.

Reply:
Nice - this gap in my knowledge is due to my overuse of the useSelector method.

Reply:
React hooks are the future! :) I also prefer the useSelector hook.